patch.py

```
@@ -0,0 +1,162 @@
+import os
+import re
+
+base = "/Users/billeaston/Documents/repos/fastforward-simd-experiment/"
+
+def patch(path, old, new):
+    with open(base + path, 'r') as f:
```

Ruff (0.15.11)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)